### PR TITLE
Add azure theme

### PR DESCRIPTION
**Pull request by micahgodbolt**
Adds Azure theme to our demo docs. Don't merge until Azure team is ready to add them.

### CODE REVIEW

File: apps/fabric-website-resources/package.json [hunk 1]
```diff
@@ -56,6 +56,7 @@
     "sinon": "^4.1.3"
   },
   "dependencies": {
+    "@uifabric/azure-themes": ">=0.1.0 <1.0.0",
```
[the remark this answers is not in the document]
Comment (micahgodbolt) [reply apps/fabric-website-resources/package.json]: I was able to install and run locally. does it actually have to be published to NPM to work?

**Comment by micahgodbolt** [comment]
Got a thumbs up from the azure team to push this out.

File: apps/fabric-website-resources/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (JasonGore) [reply apps/fabric-website-resources/package.json]: If it isn't published, anybody pulling Fabric from NPM won't be able to get azure-themes as a dependency. `azure-themes` should be published before this PR is merged.

[the remark this answers is not in the document]
Comment (JasonGore) [reply apps/fabric-website-resources/package.json]: Let me know if you want me to manually publish `azure-themes` for you. 😄

**Pull request merged**